mount/mount_linux.go

```diff
@@ -43,6 +43,8 @@ const (
 	fsckErrorsCorrected = 1
 	// 'fsck' found errors but exited without correcting them
 	fsckErrorsUncorrected = 4
+	// Log message where sensitive mount options were removed
+	sensitiveOptionsRemoved = " [sensitiveOptions Removed] "
 )
```

Comment:
where we parse from the filesystem, do we also need to stop logging content from /proc/mounts in error cases:

https://github.com/kubernetes/utils/blob/861946025e3491219eaccb1bf693e23df70c2fa8/mount/mount_linux.go#L458-L461

Reply:
Good point. Unfortunate, because if it ever fails having the actual line will be valuable debugging. Will remove and comment.